tests/test.cpp

```diff
@@ -569,7 +570,17 @@ void JsonDefaultTest() {
   // default value of the "color" field is Blue
   TEST_EQ(std::string::npos != jsongen.find("color: \"Blue\""), true);
   // default value of the "testf" field is 3.14159
-  TEST_EQ(std::string::npos != jsongen.find("testf: 3.14159"), true);
+  if(useHexfloat)
+    TEST_EQ(std::string::npos != jsongen.find("testf: 0x1.921fa0p+1"), true);
+  else
+    TEST_EQ(std::string::npos != jsongen.find("testf: 3.14159"), true);
+
```

Comment:
No need for empty lines

Reply:
Could it be better to check both representations of a pi-number in one test, without the external argument "useHexfloat"?
First test as is:
>TEST_EQ(std::string::npos != jsongen.find("testf: 3.14159"), true);

Next, regenerate json output:
>parser.opts.generate_hexfloat_in_json = true;
TEST_EQ(GenerateText(parser, builder.GetBufferPointer(), &jsongen), true);
TEST_EQ(std::string::npos != jsongen.find("testf: 0x1.921fa0p+1"), true);